packages/react/src/primitives/types/card.ts

```diff
@@ -3,5 +3,8 @@ import { ViewProps } from './view';
 export type CardVariations = 'outlined' | 'elevated';
 
 export interface CardProps extends ViewProps {
+  /**
+   * Changes the displayed style of the Card. Options include ‘outlined’, ‘elevated’ and none (default)
```

Comment:
Thank you so much for adding the descriptions!!! ❤️ 

Would you mind also add a `@description` tag in front to help `ts-morph` grab the text ?